### ISSUE TITLE
Table panel variable expansion in override malfunction

### ISSUE THREAD

**Issue by faxm0dem** [grafana/grafana#38854]
Using 8.1.2 and 8.1.1 it seems there is something wrong with variable expansion.
In the table panel, I use the following overrides for columns: fields with name `host` > `Data Links`, and with the following value:

```
/d/000000002/by-usage-and-role?orgId=1&var-site_usage=${__data.fields["facter.site_usage"]}﻿&var-site_role=All&var-site_instance=All&var-host=﻿${__data.fields.host}﻿&var-precision=$__auto_interval_precision&var-cfunc=avg
```

This used to work fine in 7.x. Now it seems to work for the first line of the table only. In other lines, the variables expand to an empty string. However, things seem normal when editing the panel.

Following video should show this more clearly (look at the link expansion on the bottom of the video). Look at variable `host`

![20210903-grafana-table-bug](https://user-images.githubusercontent.com/112638/132017875-b1899589-4f6c-4842-ad03-1c1ca7aac416.gif)

**Comment by mellieA** [comment]
@faxm0dem I'm reviewing some older issues that needed to be investigated. Are you still seeing this issue in a newer release?

**Comment by faxm0dem** [comment]
I'm still seeing this in 8.3.3

**Comment by Simonefardella** [comment]
Also the latest version (9.0.5) has this issue?

**Comment by faxm0dem** [comment]
It seems this is fixed in 9.0.5 !
